openpype/hosts/maya/plugins/publish/validate_path_attributes.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidatePathForPlugin(pyblish.api.InstancePlugin):
```

Comment:
Now we still can - should we also refactor the class name to `ValidatePluginPathAttributes`?